src/peft/peft_model.py

```diff
@@ -571,7 +645,8 @@ def forward(
         return_dict=None,
         **kwargs,
     ):
-        if not isinstance(self.peft_config, PromptLearningConfig):
+        peft_config = self.active_peft_config
```

Comment:
IMO it should be documented somewhere that the way retrieving the peft config has changed, now it's `active_peft_config` rather than `peft_config`